### DISCUSSION TITLE
audio transcription

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#1746] [Q&A]
I'm doing tests with "enableAudioTranscription" and would like to clarify some things:
1 - Is there a way to know which files were transcribed and which were not?
2 - Where can I find the transcription of the files that were transcribed?

tks, 
Paulo

**Comment by lfcnassif** [comment]
> 1 - Is there a way to know which files were transcribed and which were not?

Search for `audio\:transcription:*`
It should bring the transcribed audios.



> 2 - Where can I find the transcription of the files that were transcribed?

In that same `audio\:transcription` metadata.
